DEVELOPING.md

```diff
@@ -47,7 +47,7 @@ See [getpan/Carton.md](getpan/Carton.md) for more information.
 
 ### Things to do
 
-See the [GoPAN issues page](https://github.com/ian-kent/gopan/issues) on GitHub
+See the [GoPAN issues page](https://github.com/companieshouse/gopan/issues) on GitHub
```

Comment:
Can not currently raise issues on a fork (by default, though we could enable this if we wanted) so this link is not valid